src/extension/csw/simple-store/src/test/java/org/geoserver/csw/store/simple/SimpleCatalogStoreTest.java

```diff
@@ -72,22 +73,11 @@ public void testCapabilities() throws Exception {
     @Test
     public void testCreationExceptions() throws IOException {
         try {
-            new SimpleCatalogStore(new File("./pom.xml"));
+            new SimpleCatalogStore(Files.asResource(new File("./pom.xml")));
             fail("Should have failed, the reference is not a directory");
         } catch(IllegalArgumentException e) {
             // fine
         }
-        
-        File f = new File("./target/notThere");
-        if(f.exists()) {
-            FileUtils.deleteDirectory(f);
-        }
-        try {
-            new SimpleCatalogStore(f);
-            fail("Should have failed, the reference is not there!");
-        } catch(IllegalArgumentException e) {
-            // fine
-        }
```

Comment:
@NielsCharlier why was this test removed? You should be able to assume nonexistence and preserve this test, even though it is now testing Resource not SimpleCatalogStore. Something like:

```
new SimpleCatalogStore(Files.asResource(new File("./target/notThere")));
```